Dockerfile

```diff
@@ -11,6 +19,8 @@ RUN go build -o /prometheus-kafka-adapter
 
 FROM alpine:3.11
 
+# For Dev, Get libkafka build in previous layer (not production)
+#COPY --from=build /usr/local/lib/librdkafka.so.1 /usr/local/lib/librdkafka.so.1
 RUN echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories && \
     echo "@edgecommunity http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk add --no-cache 'librdkafka@edgecommunity>=1.3.0'
```

Comment:
I'd rather add a Dockerfile.dev file to build an image for development purposes.
However, I don't really understand why the original Dockerfile is not valid for development. It is already using the same librdkafka 1.3.0 I see you use in L7.

Reply:
Correct, this was used before you update the image, to build a local libkafka and use it in the container.